src/services/authService.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  LoginInterface,
+  LoginResponseInterface,
+  RegisterResponseInterface,
+} from "../types/authTypes";
+import { anonymPostRequest } from "./apiService";
+
+export async function registerRequest(config: LoginInterface) {
```

Comment:
Maybe it is better to call it `UserCredentialsInterfce` or smth to make it a bit more consistent?